WalletWasabi.Tests/RegressionTests/RpcServerTests.cs

```diff
@@ -0,0 +1,121 @@
+using System;
+using System.Linq;
+using System.Net;
+using System.Net.Http;
+using System.Threading;
+using System.Threading.Tasks;
+using Moq;
+using NBitcoin;
+using NBitcoin.Protocol;
+using Newtonsoft.Json;
+using Shouldly;
+using WalletWasabi.BitcoinCore.Rpc;
+using WalletWasabi.BitcoinP2p;
+using WalletWasabi.Blockchain.Analysis.FeesEstimation;
+using WalletWasabi.Blockchain.Keys;
+using WalletWasabi.Blockchain.TransactionBroadcasting;
+using WalletWasabi.Helpers;
+using WalletWasabi.Models;
+using WalletWasabi.Rpc;
+using WalletWasabi.Services;
+using WalletWasabi.Services.Terminate;
+using WalletWasabi.Stores;
+using WalletWasabi.Tests.XunitConfiguration;
+using WalletWasabi.Wallets;
+using WalletWasabi.WebClients.Wasabi;
+using Xunit;
+
+namespace WalletWasabi.Tests.RegressionTests
+{
+	[Collection("RegTest collection")]
+	public class RpcServerTests
+	{
+		private RegTestFixture RegTestFixture { get; }
+		private JsonRpcServer RpcServer { get; set; }
+
+		public RpcServerTests(RegTestFixture regTestFixture)
+		{
+			RegTestFixture = regTestFixture;
+		}
+
+		[Fact]
+		public async Task GetStatusTestAsync()
+		{
+			(string password, IRPCClient rpc, Network network, _, ServiceConfiguration serviceConfiguration, BitcoinStore bitcoinStore, Backend.Global global) = await Common.InitializeTestEnvironmentAsync(RegTestFixture, 1);
```

Comment:
Can you elaborate if Mocking can be used instead of rpc? 

https://github.com/molnard/WalletWasabi/blob/b79daf58c8aab98cd21f4bf5b0024b305cf1bf15/WalletWasabi.Tests/UnitTests/WabiSabi/Integration/WabiSabiHttpApiIntegrationTests.cs#L177

Or if you need a real Node:
https://github.com/molnard/WalletWasabi/blob/b79daf58c8aab98cd21f4bf5b0024b305cf1bf15/WalletWasabi.Tests/UnitTests/WabiSabi/Integration/WabiSabiHttpApiIntegrationTests.cs#L275

Reply:
Alright! Looking forward to it.